Assets/Scripts/MovePlayer.cs

```diff
@@ -24,19 +35,20 @@ private void Awake()
     void Start()
     {
         rgb = GetComponent<Rigidbody2D>();
-        boxCollider = GetComponent<BoxCollider2D>();
+        collider = GetComponent<BoxCollider2D>();
```

Comment:
如果不是必要，不要获取`BoxCollider2D`，而是获取`Collider2D`。

否则如果修改了玩家身上的collider类型，还得修改代码。
```suggestion
        collider = GetComponent<Collider2D>();
```